carla_ros2_agent:
  ros__parameters:
    autoware:
      camera_raw_path: "/sensing/camera/camera0/image_rect_color"
      camera_info_path: "/sensing/camera/camera0/camera_info"
      lidar_path: "/sensing/lidar/top/outlier_filtered/pointcloud"
      nav_sat_path: "/sensing/gnss/ublox/nav_sat_fix"
      ecef_velocity_twist_path: "/sensing/gnss/ecef_twist_with_covariance"
      nav_sat_orientation_path: "/autoware_orientation"
      imu_path: "/sensing/imu/tamagawa/imu_raw"
      steering_status_path: "/vehicle/status/steering_status"
      hazard_status_path: "/vehicle/status/hazard_lights_status"
      turn_indicators_status_path: "/vehicle/status/turn_indicators_status"
      gear_status_path: "/vehicle/status/gear_status"
      control_mode_status_path: "/vehicle/status/control_mode"
      veloticy_status_path: "/vehicle/status/velocity_status"
      actuation_status_path: "/vehicle/status/actuation_status"
      handbrake_status_path: "/vehicle/status/handbrake_status"
      vehicle_velocity_twist_path: "/sensing/vehicle_velocity_converter/twist_with_covariance"
      clock_path: "/clock"

    carla:
      frame_rate: 20
      actuation_cmd_path: "/control/command/actuation_cmd"
      turn_indicators_path: "/control/command/turn_indicators_cmd"
      hazard_lights_path: "/control/command/hazard_lights_cmd"
      gear_cmd_path: "/control/command/gear_cmd"
